ui/perfherder/compare/CompareTableView.jsx

```diff
@@ -59,7 +59,10 @@ export default class CompareTableView extends React.Component {
   }
 
   componentDidUpdate(prevProps) {
-    if (this.props.location.search !== prevProps.location.search) {
```

Comment:
I realized we need to provided fallback for these 3 parameters as well:
* `newProject`
* `originalProject`
* `originalRevision`

Reply:
@esanuandra please also rebase this on latest master.